lib/helper/Appium.js

```diff
@@ -0,0 +1,839 @@
+        WebDriverIO requires at least these parameters
+        Check your codeceptjs config file to ensure these are set properly
+          {
+            "helpers": {
+              "WebDriverIO": {
+                "url": "YOUR_HOST"
+                "browser": "YOUR_PREFERED_TESTING_BROWSER"
+              }
+            }
+          }
+      `);
+      }
+    } else {
+      if ((this.options.desiredCapabilities.platformName == "iOS") && ((!this.options.desiredCapabilities.app || !this.options.desiredCapabilities.deviceName || !this.options.desiredCapabilities.platformVersion) && (!this.options.desiredCapabilities.browserName) && (!this.options.desiredCapabilities.bundleId))) {
+        throw new Error(`
+          WebDriverIO requires at least these parameters to run in Appium mode for iOS
+          Check your codeceptjs config file to ensure these are set properly
+            {
+              "helpers": {
+                "WebDriverIO": {
+                  desiredCapabilities: {
+                      platformName: "iOS",
+                      app: "http://myapp.com/app.ipa",
+                      deviceName: "iPhone Simulator",
+                      platformVersion: "7.1"
+                  }
+                }
+              }
+            }
+            or
+            {
+              "helpers": {
+                "WebDriverIO": {
+                  desiredCapabilities: {
+                      platformName: "iOS",
+                      browserName: "Safari",
+                      deviceName: "iPhone Simulator",
+                      platformVersion: "7.1"
+                  }
+                }
+              }
+            }
+        `);
+      } else if ((this.options.desiredCapabilities.platformName == "Android") && ((!this.options.desiredCapabilities.appPackage || !this.options.desiredCapabilities.appActivity || !this.options.desiredCapabilities.platformVersion) && (!this.options.desiredCapabilities.browserName))) {
+        throw new Error(`
+          WebDriverIO requires at least these parameters to run in Appium mode for Android
+          Check your codeceptjs config file to ensure these are set properly
+            {
+              "helpers": {
+                "WebDriverIO": {
+                  desiredCapabilities: {
+                      platformName: "Android",
+                      appPackage: "com.aim.condition",
+                      appActivity: "com.aim.condition.activities.MainActivity",
+                      platformVersion: "6.0.1"
+                  }
+                }
+              }
+            }
+          or
+          {
+            "helpers": {
+              "WebDriverIO": {
+                desiredCapabilities: {
+                    platformName: "Android",
+                    browserName: "Chrome",
+                    platformVersion: "6.0.1"
+                }
+              }
+            }
+          }
+        `);
+      }
+    }
+
+  }
+
+
+  _startBrowser() {
```

Comment:
If we can add browserCap check in `WebdriverIO` helper 
```
if (this.options.timeouts && browserCap) {
      this.defineTimeout(this.options.timeouts);
    }
```
then we will not have this duplicate.

The same thing with `_after` hook